evol/population.py

```diff
@@ -10,13 +10,14 @@
 from itertools import cycle, islice
 
 from evol import Individual
-from evol.helpers.utils import select_arguments, offspring_generator
+from evol.helpers.utils import select_arguments, offspring_generator, ensure_list
 
 
 class Population:
     """Population of Individuals
 
     :param chromosomes: Collection of initial chromosomes of the Population.
+    #TODO What is this Collection? Should it be a typing.List, or perhaps a typing.Sequence?
```

Comment:
I think `typing.Sequence` should be fine. I can't think of a reason why you couldn't give us a large tuple. @rogiervandergeer ?

Reply:
Yes, should be `Sequence`